src/plugins/github/relationalView.js

```diff
@@ -393,7 +395,7 @@ export class RelationalView {
     //   https://github.com/sourcecred/sourcecred/pull/416
     // - A # followed by a number, such as #416
     // - An @ followed by a login name, such as @decentralion
-    const refToAddress: Map<string, N.StructuredAddress> = new Map();
+    const refToAddress: Map<string, N.ReferentAddress> = new Map();
```

Comment:
No objection to this change. Consider mentioning in the commit message.

Reply:
Done.